_data/canonical-tags/networking.yaml

```diff
@@ -0,0 +1,3 @@
+id: networking
+name: Networking
+description: Relevant to how Kubernetes components talk to each other (and to programs outside the cluster).
```

Comment:
> How Kubernetes components talk to each other and to programs outside the cluster.

Reply:
:+1: I'll be changing all of these as you suggested, but want to provide some context. Initially I prefaced a lot of these terms with "relevant to" because the description shows up as follows:
<img width="1092" alt="screen shot 2017-10-10 at 8 57 42 am" src="https://user-images.githubusercontent.com/25401650/31396742-1e3ef050-ad99-11e7-85f0-be055ceec424.png">

I wasn't sure if the description should directly apply to the tags themselves (in which case, it's clear that "relevant for" should be omitted), or to the *terms* described by the tags---so I initially opted for the latter.

Reply:
Thanks for the context, it helps! 👍